demo/lightning/checkpoint/simulated/multiprocessing_train.py

```diff
@@ -235,7 +310,7 @@ def main():
     args = parse_args()
 
     mp.set_start_method('spawn')
-    mp.spawn(run_benchmark, args=(args.world_size, args.layer_size, args.project, args.ckpt_dir_path, args.padding_size, args.sample_count),
+    mp.spawn(run_benchmark, args=(args.world_size, args.layer_size, args.project, args.ckpt_dir_path, args.padding_size, args.sample_count, args.debug),
```

Comment:
Based on this formatting I don't think you're VS code is setup with the correct formatter. We are using the vscode settings specified in the `.vscode` dir in the root of this project. Can you double check that those are configured and active? Once made active you should be able to just save and have it reformat automatically in one pass.

Reply:
Just synced the formatting